### PR TITLE
Update Alumni List

### PR DESCRIPTION
**Pull request by yardasol**
## Summary of changes

This PR moves some folks who are no longer active in the group anymore to the Alumni page.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Developers

- Dev: @nsryan2 

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests.md) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Review by nsryan2 — APPROVED** [review]
LGTM!

File: _data/alumni.yml [hunk 1]
```diff
@@ -476,3 +476,31 @@
...
+  pic: /img/people/pacioccog.png
+  dates: 2021-2022
+  role: Undergraduate Research Assistant 
+  description: 'Gabriele is an NPRE undergraduate at UIUC. His interests in Math and Science motivate him to study the future of nuclear reactor devlopement.'
+  social:
+    - url: mailto:Agp9@illinois.edu
+      icon: envelope-o
+      name: Email
+    - url: https://www.linkedin.com/in/gabriele-paciocco-b311b41a1/
+      icon: linkedin
```
Comment (munkm) [inline _data/alumni.yml:503]: Is this case-sensitive? The `icon:` on l477 has it capitalized?

**Review by munkm — APPROVED** [review]
Looks good to me! I had one minor question but it isn't blocking.

File: _data/alumni.yml (hunk 1 above)
Comment (nsryan2) [reply _data/alumni.yml:503]: It looks like he got it from my profile which doesn't have a capitalized L, but other do and it renders on the site so I think we're good either way

**Pull request merged**